query-planner-js/src/composedSchema/core.ts

```diff
@@ -0,0 +1,32 @@
+import { URL } from "apollo-server-env";
```

Comment:
Not asking for a change, but I'd be surprised if we were in an environment where `URL` wasn't available.   I suppose raw V8, which I'm not sure if this works on yet.

Reply:
Good catch, this import is technically borked since this package isn't listed as a dependency. Switching over to import from node native `url`.

Reply:
LGTM.  Happy to revisit later.